mkdocs.yml

```diff
@@ -67,6 +67,7 @@ pages:
       - Built-in Chat Integrations: lifecycle.md
       - Spring: user/spring.md
       - Kubernetes: user/kubernetes.md
+      - Supported Integrations: integrations.md
```

Comment:
Should this be under user/ so the URLs are consistent?

Reply:
yes, great point.